config/initializers/paranoia.rb

```diff
@@ -0,0 +1,15 @@
+module Paranoia
```

Comment:
I'd deal with it after successfully upgrading. It'll definitely make the next upgrade harder, at least.

Reply:
I think this bug only affects the test suite. It could be that we can remove this after we merge this branch and upgrade Capybara / Selenium, but we'll have to see.